MOBILE_CLIENT/COMMON/_HIGH/XPATH_INJECTION/recommendation.md

```diff
@@ -0,0 +1,91 @@
+
+# Xpath Injection
+
+To mitigate XPath Injection vulnerabilities, it is important to properly validate and sanitize user input before using it in XPath queries. This can be done by using parameterized queries or prepared statements, which separate the user input from the query logic. Additionally, limiting the privileges of the user executing the query can help prevent unauthorized access to sensitive data. Regularly updating and patching software and libraries can also help prevent known vulnerabilities from being exploited. Finally, implementing logging and monitoring can help detect and respond to any attempted attacks.
```

Comment:
There parametrized queries on xpath?

Reply:
depends on the package but yes, as far as I understand